src/modules/home/Trending.tsx

```diff
@@ -1,8 +1,8 @@
 import { useQuery } from "@tanstack/react-query";
-import { IManga } from "../types/app-types";
-import { ITrendingManga } from "../types/subtypes";
-import useMangaStore from '../stores/store'
-import { fetchTrendingMangas } from "../services/api-service";
+import { ITrendingMangaApp } from "../common/types/app-types";
+import { ITrendingManga } from "../common/types/subtypes";
+import useMangaStore from '../common/stores/store'
+import { fetchTrendingMangas } from "../common/services/api-service";
 
 function Trending() {
 
```

Comment:
```Line 9```: Check the naming convention of ```TopComicResponse```, I believe this should be camelCase.

Reply:
Good eye, much appreciated!